mappings/net/minecraft/world/ChunkSerializer.mapping

```diff
@@ -24,7 +24,7 @@ CLASS net/minecraft/class_2852 net/minecraft/world/ChunkSerializer
 		ARG 1 structureManager
 		ARG 2 poiStorage
 		ARG 3 pos
-		ARG 4 tag
+		ARG 4 nbt
 	METHOD method_12396 (Lnet/minecraft/class_2248;)Z
 		ARG 0 block
 	METHOD method_12410 serialize (Lnet/minecraft/class_3218;Lnet/minecraft/class_2791;)Lnet/minecraft/class_2487;
```

Comment:
Maybe `toNbt`. This class has an interesting context; the `read` methods are technically fromNbt ones, but seems in this context `read` without `Nbt` works better than `fromNbt`. Needs checking.